### PR TITLE
Calculated TVL amount

### LINKED ISSUE #140: Calculate TVL amount
1. Read [guidelines](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5).
2. We're fetching amount for all opened position [here](https://github.com/djeck1432/spotnet/blob/main/web_app/db/crud.py#L365) but we sum all position together despite on the token for each position, however, there could be different tokens, like: `ETH`, `STRK`, `USDC`.
3. Fetch current prices using this method [get_current_prices](https://github.com/djeck1432/spotnet/blob/main/web_app/contract_tools/mixins/dashboard.py#L31)
4. Map current prices and amount for each token position and then return in this method[get_stats](https://github.com/djeck1432/spotnet/blob/main/web_app/api/user.py#L138) sum of all assets based on current price:
p.s. so if you have in db `ETH`: 100, `STRK`: 20, `USDC`: 100, you should convert `ETH` and `STRK` to `USDC` current price and multiply on the amount of these tokens appropriate and return total amount of all assets in `USDC`

### PR DESCRIPTION
**Pull request by josephchimebuka**
fix #140

### CODE REVIEW

File: web_app/api/user.py [hunk 1]
```diff
@@ -135,22 +135,48 @@ async def get_user_contract_address(wallet_id: str) -> GetUserContractAddressRes
...
+        
         unique_users = user_db.get_unique_users_count()
+        
         return GetStatsResponse(
-            total_opened_amount=total_opened_amount, unique_users=unique_users
+            total_opened_amount=total_opened_amount, 
+            unique_users=unique_users
         )
-
-    except AttributeError as e:
```
Comment (djeck1432) [inline web_app/api/user.py:153]: Why did you remove this code?

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: web_app/api/user.py (hunk 1 above)
Comment (josephchimebuka) [reply web_app/api/user.py:153]: It felt redundant that was why.I thought this was closed already that why I am responding now

**Review by djeck1432 — APPROVED** [review]
Looks good to me
